modules/apps/data-engine/data-engine-taglib/src/main/java/com/liferay/data/engine/taglib/servlet/taglib/DataLayoutBuilderTag.java

```diff
@@ -177,15 +173,9 @@ private ServletContext _getModuleServletContext() {
 	}
 
 	private String _getPluginEntryPoint(String value) {
-		if (DataLayoutTaglibUtil.isUsingNewFormProvider()) {
-			return DataLayoutTaglibUtil.resolveModule(
-				"data-engine-taglib/data_layout_builder/new-js/plugins/" +
-					value + "/index");
-		}
-
 		return DataLayoutTaglibUtil.resolveModule(
-			"data-engine-taglib/data_layout_builder/js/plugins/" + value +
-				"/index.es");
+			"data-engine-taglib/data_layout_builder/new-js/plugins/" + value +
```

Comment:
Do you guys have a plan to move the content from the `new-js/` folder to `js/`?

Reply:
After remove the feature flag we should do a cleanup on the code to remove the old stuff and merge both into the js.